arch/HPPA/HPPADisassembler.c

```diff
@@ -1,7 +1,7 @@
 /* Capstone Disassembly Engine */
 /* By Dmitry Sibirtsev  <sibirtsevdl@gmail.com>, 2023 */
 
-#ifdef CAPSTONE_HAS_HPPA
+// #ifdef CAPSTONE_HAS_HPPA
```

Comment:
In the `CMakeList.txt` there is a variable with all arch names `SUPPORTED_ARCHITECTURES` and `SUPPORTED_ARCHITECTURES_LABELS`. You can add HPPA there and the macro is generated during build.